pkgs/development/python-modules/dalle-mini/default.nix

```diff
@@ -24,8 +25,20 @@ buildPythonPackage rec {
 
   format = "setuptools";
 
-  buildInputs = [
-    jaxlib
+  # Those two patches are caused by changes in the latest jax versions
+  patches = [
+    ./fix-partitionspec-import.patch
```

Comment:
Please add links to the details of the patches, if possible. Without pointing to the upstream issues it's quite difficult to track the patches.

Reply:
The devs have decided to pin jax. As we cannot do that I went with writing this patch myself.
I made a better comment though. Is that OK for you ?